languageserver/server/server.go

```diff
@@ -145,6 +145,9 @@ type AddressContractNamesResolver func(address common.Address) ([]string, error)
 // StringImportResolver is a function that is used to resolve string imports
 type StringImportResolver func(location common.StringLocation) (string, error)
 
+// IdentifierImportResolver is a function that is used to resolve identifier imports supported by flowkit
+type IdentifierImportResolver func(location common.IdentifierLocation) (string, error)
+
```

Comment:
Looks like this isn't necessary anymore:

```suggestion
```

Reply:
damn